Thermometer crash reported in Google Play Store

**Actual Behaviour**

Google Play Device testing has reported an issue in Thermometer crashing when selecting SHT sensor in thermometer

**Expected Behaviour**

No crashes

**Steps to reproduce it**

1. Open app
2. Go to Thermometer
3. Open configurations
4. Set sensor to SHT21 sensor

**LogCat for the issue**
```
FATAL EXCEPTION: ControllerMessenger
Process: io.pslab, PID: 28552
java.lang.RuntimeException: Unable to resume activity {io.pslab/io.pslab.activity.ThermometerActivity}: java.lang.ArrayIndexOutOfBoundsException: length=2; index=2
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3788)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3828)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1746)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at androidx.test.espresso.base.Interrogator.a(Interrogator.java:19)
	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:166)
	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:158)
	at androidx.test.espresso.base.UiControllerImpl.a(UiControllerImpl.java:34)
	at androidx.test.espresso.action.MotionEvents.a(MotionEvents.java:77)
	at androidx.test.espresso.action.MotionEvents.a(MotionEvents.java:52)
	at androidx.test.espresso.action.Tap.c(Tap.java:8)
	at androidx.test.espresso.action.Tap.b(Tap.java:18)
	at androidx.test.espresso.action.Tap$1.a(Tap.java:3)
	at androidx.test.espresso.action.GeneralClickAction.perform(GeneralClickAction.java:22)
	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAction.perform(ViewInteraction.java:9)
	at androidx.test.espresso.ViewInteraction.a(ViewInteraction.java:79)
	at androidx.test.espresso.ViewInteraction.a(ViewInteraction.java:96)
	at androidx.test.espresso.ViewInteraction$1.call(ViewInteraction.java:3)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.Handler.handleCallback(Handler.java:789)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6938)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
Caused by: java.lang.ArrayIndexOutOfBoundsException: length=2; index=2
	at io.pslab.fragment.ThermometerDataFragment.initiateThermoSensor(ThermometerDataFragment.java:559)
	at io.pslab.fragment.ThermometerDataFragment.onResume(ThermometerDataFragment.java:162)
	at android.support.v4.app.Fragment.performResume(Fragment.java:2498)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1501)
	at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
	at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(FragmentManager.java:3269)
	at android.support.v4.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:3241)
	at android.support.v4.app.FragmentController.dispatchResume(FragmentController.java:223)
	at android.support.v4.app.FragmentActivity.onResumeFragments(FragmentActivity.java:538)
	at android.support.v4.app.FragmentActivity.onPostResume(FragmentActivity.java:527)
	at android.support.v7.app.AppCompatActivity.onPostResume(AppCompatActivity.java:172)
	at android.app.Activity.performResume(Activity.java:7368)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3763)
```

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

GSoC issue

--- a/app/src/main/java/io/pslab/fragment/ThermometerDataFragment.java
+++ b/app/src/main/java/io/pslab/fragment/ThermometerDataFragment.java
@@ -556,7 +556,7 @@ public class ThermometerDataFragment extends Fragment {
                 }
                 break;
             case SHT21_SENSOR:
-                sensorLabel.setText(getResources().getStringArray(R.array.thermo_sensors)[2]);
+                sensorLabel.setText(getResources().getStringArray(R.array.thermo_sensors)[1]);
                 scienceLab = ScienceLabCommon.scienceLab;
                 if (scienceLab.isConnected()) {
                     try {

